posthog/caching/calculate_results.py

```diff
@@ -29,6 +29,11 @@
 from posthog.queries.trends.trends import Trends
 from posthog.types import FilterType
 
+# ClickHouse query timeout in seconds
+# From https://github.com/PostHog/posthog-cloud-infra/blob/master/ansible/config/clickhouse-users.xml#L11
+# Keep in sync with the above!
```

Comment:
This comment needs to be added into the XML as well, otherwise this will not stay in sync :).

We'll find out quickly if it ever does go out of sync though, so 🤷